content/diffPage.js

```diff
@@ -181,7 +198,11 @@ _writeViewFrame : function (view, url, enableDiffLinks)
         viewDoc.write("<span style='color:#808080'>"+changes+"</span>\n&nbsp;");
 
 
-    viewDoc.write("<a href='"+url+"' target='_top'>"+currentPage+"</a>\n");
+    if (postParams) {
+        viewDoc.write("<a href='javascript:void(0);' onclick='document.currentPageForm.submit();'>"+currentPage+"</a>\n");
```

Comment:
Because of this reason alone, I decided to refactor my modifications to the entire file, I'll push what I've made soon.